arcgis-ios-sdk-samples/Edit data/Offline edit and sync/OfflineEditingViewController.swift

```diff
@@ -234,9 +234,12 @@ class OfflineEditingViewController: UIViewController, AGSMapViewTouchDelegate, A
                 self?.liveMode = false
                 
                 self?.map.operationalLayers.removeAllObjects()
-                for featureTable in geodatabase.geodatabaseFeatureTables {
-                    let featureLayer = AGSFeatureLayer(featureTable: featureTable)
-                    self?.map.operationalLayers.addObject(featureLayer)
+                for (_, featureTable) in geodatabase.geodatabaseFeatureTables.enumerate().reverse() {
```

Comment:
If you dont need the index value (_), then wouldn't it be simpler to just do `for featureTable in gdb.geodatabaseFeatureTables.reverse()` instead of using enumerate().reverse() ?